app/src/main/java/de/danoeh/antennapod/fragment/actions/FeedMultiSelectActionHandler.java

```diff
@@ -126,6 +126,33 @@ private void keepUpdatedPrefHandler() {
         preferenceSwitchDialog.openDialog();
     }
 
+    private void newEpisodesActionPrefHandler() {
```

Comment:
Is this method ever called?

Reply:
It doesn't look like it is. I was using the "Auto Delete" option as a starting point to figure out how to add this option, and so I implemented this function because Auto Delete had a counterpart in this file.

Do you think an option for this action should be added to the multi-select for the "Subscriptions" screen? Might be useful, and then there would be something that uses this function.

Reply:
I guess it would make sense to add it there. I'm not entirely convinced by the ever growing list of individual multi-select settings, but changing that is something for another PR :)